### PR TITLE
[Docs] Add changes in UPGRADE-1.8.md

### PR DESCRIPTION
**Pull request by Tomanhez**
| Q               | A
| --------------- | -----
| Branch?         | 1.8
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

### CODE REVIEW

**Review by pamil — APPROVED** [review]

File: UPGRADE-1.8.md [hunk 1]
```diff
@@ -36,6 +42,10 @@
     +                   providers: [sylius_api_shop_user_provider, sylius_api_admin_user_provider]
             
             firewalls:
+                admin:
+                    remember_me:
```
Comment (lchrusciel) [inline UPGRADE-1.8.md:46]:
```suggestion
    +          admin:
    +              remember_me:
```

File: UPGRADE-1.8.md [hunk 2]
```diff
@@ -74,15 +84,53 @@
     +               guard:
     +                   authenticators:
     +                       - lexik_jwt_authentication.jwt_token_authenticator
+    + 
+            access_control:
```
Comment (lchrusciel) [inline UPGRADE-1.8.md:88]:
```suggestion
    +      access_control:
```

**Review by lchrusciel — APPROVED** [review]

File: UPGRADE-1.8.md (hunk 1 above)
Comment (Tomanhez) [reply UPGRADE-1.8.md:46]: Keys `admin `, `remember_me ` and `access_control` already exists, I think that we should and `+` to new lines

**Comment by jacquesbh** [comment]
The dynamic admin path is incorrect in the configurations diffs.

File: UPGRADE-1.8.md [hunk 3]
```diff
@@ -20,10 +20,16 @@
 
     ```diff
         parameters:
+    -       sylius.security.admin_regex: "^/admin"
     -       sylius.security.shop_regex: "^/(?!admin|api/.*|api$|media/.*)[^/]++"
+    +       sylius.security.admin_regex: "^/%sylius_admin.path_name%"
     +       sylius.security.shop_regex: "^/(?!admin|new-api|api/.*|api$|media/.*)[^/]++"
```
Comment (jacquesbh) [inline UPGRADE-1.8.md:26]:
```suggestion
    +       sylius.security.shop_regex: "^/(?!%sylius_admin.path_name%|new-api|api/.*|api$|media/.*)[^/]++"
```

**Review by jacquesbh — CHANGES_REQUESTED** [review]

File: UPGRADE-1.8.md [hunk 4]
```diff
@@ -74,15 +84,53 @@
...
     ```
-
 1. Add `sylius_api.yaml` file to `config/routes/` directory:
 
     ```yaml
        sylius_api:
            resource: "@SyliusApiBundle/Resources/config/routing.yml"
            prefix: "%sylius.security.new_api_route%"
     ```
+1. Add `lexik_jwt_authentication.yaml` file to `config/packages/` directory:
```
Comment (GSadee) [inline UPGRADE-1.8.md:99]:
```suggestion

1. Add `lexik_jwt_authentication.yaml` file to `config/packages/` directory:
```

File: UPGRADE-1.8.md [hunk 5]
```diff
@@ -74,15 +84,53 @@
...
+         pass_phrase: '%env(JWT_PASSPHRASE)%'
+    ```
+
+1. Add configuration in your `.env` file:
+
+    ```diff
+    +       ###> lexik/jwt-authentication-bundle ###
+    +       JWT_SECRET_KEY=%kernel.project_dir%/config/jwt/private.pem
+    +       JWT_PUBLIC_KEY=%kernel.project_dir%/config/jwt/public.pem
+    +       JWT_PASSPHRASE=cd093de0e84512a4087839ef05088ffe
```
Comment (GSadee) [inline UPGRADE-1.8.md:114]: There should be probably information about generating these files and that `JWT_PASSPHRASE` value is an example

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Tomasz! :1st_place_medal:

**Pull request merged**